### ISSUE TITLE
Add Syntax Highlighting

### ISSUE THREAD

**Issue by callym** [massivedanger/protagonist#7]
Adding (javascript) syntax highlighting in the editor would make the code a lot easier to read.

Harlowe has somehow managed it, I think the relevant code is [here](https://bitbucket.org/_L_/harlowe/src/3520ab2e0a7a3e52885da3946e29ee064422c7b6/js/markup/codemirror/mode.js?at=default).

**Comment by evanwalsh** [comment]
Yeah, that'd be awesome to add! I didn't even know formats could affect the editor.

**Comment by X-Raym** [comment]
@evanwalsh Hi ! Any news about this feature implementation ? It would be great to have it :)

**Comment by X-Raym** [comment]
Discussion here :
[Color Synthax HighLight to Custom Story Format - Twine Forum](http://twinery.org/forum/discussion/8353/color-synthax-highlight-to-custom-story-format)

**Comment by evanwalsh** [comment]
No news regarding this. I'm not using Twine these days, so I haven't spent much time with Protagonist. You're welcome to give it a go, though!

**Comment by X-Raym** [comment]
@evanwalsh thanks for your answer ! I tried but I was stuck at the "integrated the color syntax" steps. Seems a bit too complicated for me at this moment. So, if anyone else want to try... feel free to do it !
